Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "ValueHistories"
+uuid = "98cad3c8-aec3-5f06-8e41-884608649ab7"
+authors = ["JuliaML and contributors"]
+version = "0.5.2"
+
+[deps]
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
+RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
+
+[compat]
+DataStructures = "0.5, 0.6, 0.7, 0.8, 0.9, 0.10, 0.11, 0.12, 0.13, 0.14, 0.15, 0.16"
+RecipesBase = "0.2.3, 0.3, 0.4, 0.5"
+julia = "1"
+
```

Comment:
Is it intentional that both of these packages are capped below their current releases?

Reply:
It was not intentional. I just looked at Travis to see what the latest versions tested on Travis were. I did not realize those weren't the latest releases.

Reply:
@oxinabox Do you think I should add compat with the latest releases? I'm happy to do so.

Reply:
yes, i think it would be good to.

It was probably locked down when all things without upper bounds were capped to the latest release of each thing about when we  changed over from using METADATA at all.

Reply:
I'll make a PR to install CompatHelper on this repo. That should take care of this,